browser/about_flags.cc

```diff
@@ -479,6 +495,33 @@ constexpr char kPlaylistDescription[] = "Enables Playlist";
 #define PLAYLIST_FEATURE_ENTRIES
 #endif
 
+#if defined(TOOLKIT_VIEWS)
+uint16_t AllowForDevVersion(uint16_t os) {
+#if BUILDFLAG(IS_WIN)
+  if (!install_static::InstallDetails::GetPayload()) {
+    // During unit test, this can happen.
+    return 0;
+  }
+#endif
```

Comment:
Yay, this seems to work. tested on test-release bot

Reply:
Now it looks working! - ci test-release bot passed. @bridiver What do you think about this? I like what you suggested from https://github.com/brave/brave-core/pull/13816/files#r934765872 but we might need this too in order to hide the flag from "brave://flags" page. I guess that's why folks have touched unexpire_flags.

Reply:
this is only affecting the display of the flag, not the enable/disable state

Reply:
Yeah, that's intended. I didn't intend to change default "enable/disable" state per channels. Default state should be "disabled" for all channel before this feature get stable. And allowing to enable this feature only on Beta and Nightly is what we want here.

Reply:
@simonhong Do you think it's worth hiding this flag from stable channel? If we don't find a decent way to hide this, I think showing the flag is okay, as the default value is disabled.

Reply:
I don't think trying to hide things from channels this way makes sense. Why would we hide the flag? We can control whether it's enabled or disabled by default through griffin and if people want to change the flag then they are volunteering to beta test it

Reply:
same with @bridiver . If this is testable, it seems ok to displaying on all channels.